dojo/unittests/test_apiv2_methods.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework.test import APIRequestFactory
+from rest_framework.views import APIView
+from rest_framework.test import APITestCase, force_authenticate
+from drf_yasg.generators import OpenAPISchemaGenerator
+from drf_yasg.openapi import Info
```

Comment:
could you change it to use https://github.com/tfranzel/drf-spectacular introduced in https://github.com/DefectDojo/django-DefectDojo/pull/4541 as drf-yasg will be deprecated in 2.0.0